packages/amplify-graphql-auth-transformer/src/__tests__/__snapshots__/rds-subscription-auth.test.ts.snap

```diff
@@ -0,0 +1,2203 @@
+}))
+#set( $authResult = $util.authRules.subscriptionAuth($authRules) )
+#if( !$authResult || ($authResult && !$authResult.authorized) )
+$util.unauthorized
+#end
+#if( $authResult && !$util.isNullOrEmpty($authResult.authFilter) )
+  #if( $util.isNullOrEmpty($ctx.args.filter) )
+    #set( $ctx.args.filter = $authResult.authFilter )
+  #else
+    #set( $ctx.args.filter = {
+  \\"and\\":   [$authResult.authFilter, $ctx.args.filter]
+} )
+  #end
+#end
+## [End] Authorization rules. **"
+`;
+
+exports[`Verify RDS Model level Auth rules on subscriptions: should successfully transform cognito auth rules 26`] = `
+"## [Start] Sandbox Mode Disabled. **
+#if( !$ctx.stash.get(\\"hasAuth\\") )
+  $util.unauthorized()
+#end
+$util.toJson({})
+## [End] Sandbox Mode Disabled. **"
+`;
+
+exports[`Verify RDS Model level Auth rules on subscriptions: should successfully transform cognito auth rules 27`] = `
+"## [Start] Subscription Response template. **
+#if( !$util.isNullOrEmpty($ctx.args.filter) )
+$extensions.setSubscriptionFilter($util.transform.toSubscriptionFilter($ctx.args.filter))
+#end
+$util.toJson(null)
+## [End] Subscription Response template. **"
+`;
+
+exports[`Verify RDS Model level Auth rules on subscriptions: should successfully transform cognito auth rules 28`] = `
+"## [Start] Authorization rules. **
+$util.qr($ctx.stash.put(\\"hasAuth\\", true))
+#set( $authRules = [] )
+$util.qr($authRules.add({
+  \\"type\\": \\"groups\\",
+  \\"provider\\": \\"userPools\\",
+  \\"allowedGroups\\":   [\\"Admin\\"],
+  \\"identityClaim\\": \\"cognito:groups\\",
+  \\"allowedFields\\":   [\\"id\\", \\"title\\"]
+}))
+$util.qr($authRules.add({
+  \\"type\\": \\"groups\\",
+  \\"provider\\": \\"userPools\\",
+  \\"allowedGroups\\":   [\\"Moderator\\"],
+  \\"identityClaim\\": \\"cognito:groups\\",
+  \\"allowedFields\\":   [\\"id\\", \\"title\\"]
+}))
+#set( $authResult = $util.authRules.subscriptionAuth($authRules) )
+#if( !$authResult || ($authResult && !$authResult.authorized) )
+$util.unauthorized
+#end
+#if( $authResult && !$util.isNullOrEmpty($authResult.authFilter) )
+  #if( $util.isNullOrEmpty($ctx.args.filter) )
+    #set( $ctx.args.filter = $authResult.authFilter )
+  #else
+    #set( $ctx.args.filter = {
+  \\"and\\":   [$authResult.authFilter, $ctx.args.filter]
+} )
+  #end
+#end
+## [End] Authorization rules. **"
+`;
+
+exports[`Verify RDS Model level Auth rules on subscriptions: should successfully transform cognito auth rules 29`] = `
+"## [Start] Sandbox Mode Disabled. **
+#if( !$ctx.stash.get(\\"hasAuth\\") )
+  $util.unauthorized()
+#end
+$util.toJson({})
+## [End] Sandbox Mode Disabled. **"
+`;
+
+exports[`Verify RDS Model level Auth rules on subscriptions: should successfully transform cognito auth rules 30`] = `
+"## [Start] Subscription Response template. **
+#if( !$util.isNullOrEmpty($ctx.args.filter) )
+$extensions.setSubscriptionFilter($util.transform.toSubscriptionFilter($ctx.args.filter))
```

Comment:
nit: general comment for the VTL generated in multiple of these snapshots is that the indentation is off.

Reply:
These are generated by mapping-template helpers and jest. Not much we can do in the scope of this PR.